backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -525,7 +494,7 @@ export class UserResolver {
           `klicktippSignIn(${userContact.email}, ${user.language}, ${user.firstName}, ${user.lastName})`,
         )
       } catch (e) {
-        logger.error('Error subscribe to klicktipp:' + e)
+        logger.error(`Error subscribe to klicktipp: ${e}`, e)
```

Comment:
```suggestion
        logger.error('Error subscribe to klicktipp:', e)
```

Reply:
i chose `logger.error('Error subscribing to klicktipp', e)`